File: java/java-arithmetic/src/main/java/com/leetcode/first/J28.java
package com.leetcode.first;

public class J28 {

    public static void main(String[] args) {

        System.out.println(new J28().strStr("abc","c"));
    }

    public int strStr(String haystack, String needle) {
        if (haystack.equals(needle)){
            return 0;
        }
        for (int i=0;i<haystack.length();i++){
            if (haystack.startsWith(needle, i)){
                return i;
            }
        }
        return -1;
    }
}

File: python-test/leetcode/J28.py
class Solution:
    def strStr(self, haystack: str, needle: str) -> int:
        if haystack == needle:
            return 0
        for i in range(len(haystack)):
            print(haystack[i:i+len(needle)])
            if haystack[i:i+len(needle)] == needle:
                return i
        
        return -1


print(Solution().strStr("hello","ll"))